pages/common/git-stash.md

```diff
@@ -4,24 +4,24 @@
 
 - Stash current changes (except new files):
 
-`git stash save {{optional_stash_name}}`
+`git stash save {{optional_stash_message}}`
 
 - Include new files in the stash (leaves the index completely clean):
 
-`git stash save -u {{optional_stash_name}}`
+`git stash save -u {{optional_stash_message}}`
 
 - List all stashes:
```

Comment:
Suggestion - "List all stash names". Because the following descriptions mention "optional_stash_name". One might be confused from where to retrieve this stash name.